18F/identity-idp
# LG-6160: Add personal key validation behavior

**Why**: As a user, I expect feedback if I have incorrectly entered the personal key, so that I can correct my mistakes and continue with the proofing flow.

**Testing Instructions:**

1. Set `idv_api_enabled: "true"` in local `config/application.yml`
2. Go to: http://localhost:3000/verify/v2/personal_key_confirm
3. Click "Submit"
4. Observe error "You've entered an incorrect personal key", and that focus is returned to the field
5. Enter some text
6. Observe that the error is removed after input, consistent with other IdP inputs
7. Enter a correctly-formatted, but incorrect personal key (e.g. "0000-0000-0000-0001")
8. Click "Submit"
9. Observe error "You've entered an incorrect personal key", and that focus is returned to the field
10. Enter the correct personal key "0000-0000-0000-0000"
11. Click "Submit"
12. Observe there is no error message

**Screenshot:**

![image](https://user-images.githubusercontent.com/1779930/164045992-d91dabbc-bb07-4229-b08f-29336596ede9.png)

**Draft Progress:**

- [ ] Improve test coverage
